src/digraph.rs

```diff
@@ -2606,6 +2606,43 @@ impl PyDiGraph {
         edges.is_empty()
     }
 
+    /// Make edges in graph symmetric
+    ///
+    /// This function iterates over all the edges in the graph and for each edge if a reverse
+    /// edge is not present in the graph one will be added.
```

Comment:
Following is perhaps a bit more concise:
"This function iterates over all edges in the graph, adding for each edge the reversed edge, unless it is already present."

Reply:
Fixed in: https://github.com/Qiskit/rustworkx/pull/814/commits/acae85f05623956f2cc11b928337205a8d641aba